### PR TITLE
feat: persistence

### PR DESCRIPTION
**Pull request by aulneau**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/blockstack/ux/actions/runs/1230286375).

With the release of 1.3.1 of jotai, we can now make use of the persistence features react-query has. This makes it so that when a user re-opens the wallet, they start from the last state they had, and queries will automatically refresh as needed.

cc/ @aulneau @kyranjamie @fbwoolf

### CODE REVIEW

**Comment by aulneau** [comment]
@beguene @kyranjamie @fbwoolf can I get a review on this? would love to shipit

File: webpack/webpack.config.base.js [hunk 1]
```diff
@@ -205,7 +205,7 @@ const config = {
       'process.env.TEST_ENV': JSON.stringify(TEST_ENV ? 'true' : 'false'),
     }),
     new webpack.ProvidePlugin({
-      process: 'process/browser',
+      process: 'process/browser.js',
```
Comment (kyranjamie) [inline webpack/webpack.config.base.js:208]: how come?

**Review by kyranjamie — APPROVED** [review]
is 14.x a legit input for the node versions?

File: webpack/webpack.config.base.js (hunk 1 above)
Comment (aulneau) [reply webpack/webpack.config.base.js:208]: ESM/CJS issues

**Comment by aulneau** [comment]
@timstackblock @Eshwari007 can you QA this today? It's mostly a performance enhancement, so things like opening it after the initial load should seem faster, and generally there should be no performance regressions.

**Comment by Eshwari007** [comment]
@aulneau Yes will work on it and keep you posted.Thx

**Comment by Eshwari007** [comment]
Performance has improved greatly and no spinning issues when user selects accounts with various tokens/accounts
cc @aulneau

**Comment by andresgalante** [comment]
@Eshwari007 for next time, I wouldn't move a PR to Done if it hasn't been merged yet.

@aulneau Please let us know when the conflict is resolved so we can merge it

**Pull request merged**